lib/bridge/github/contents.rb

```diff
@@ -23,7 +23,7 @@ def issue_template_content
         content = nil
       end
 
-      return content
+      return content ? content.force_encoding("utf-8") : content
```

Comment:
So the difference is that `""` would be switched to `nil` using the latter?

Doesn't really matter, just learning what idioms work when.